File: Python/monkey_trouble.py
def monkey_trouble(a_smile, b_smile):
    if a_smile == True and b_smile == True:
        return True
    elif a_smile == False and b_smile == False:
        return True
    else:
        return False


if __name__ == '__main__':
    print(monkey_trouble(True, True))  # True
    print(monkey_trouble(False, False))  # True
    print(monkey_trouble(True, False))  # False

File: MonkeyTrouble.java

public class MonkeyTrouble {

	public static void main(String[] args) {

		System.out.println(monkeyTrouble(true, true)); // true
		System.out.println(monkeyTrouble(false, false)); // true
		System.out.println(monkeyTrouble(true, false)); // false
	} // end main method

	// We have two monkeys, a and b, and the parameters aSmile and bSmile indicate
	// if each is smiling.
	// We are in trouble if they are both smiling or if neither of them is smiling.
	// Return true if we are in trouble.

	public static boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
		if ((aSmile && bSmile) || (!aSmile && !bSmile)) {
			return true;
		}
		return false;
	} // end monkey method

} // end class
